src/cloud_common/protocols/unit/unit_config.py

```diff
@@ -152,6 +152,15 @@ class ResourceRatiosInfo(BaseModel):
         ),
     ]
 
+    state: Annotated[
+        float,
+        Field(
+            default=None,
+            alias='state',
+            description='The state ratio in percent.',
```

Comment:
Please improve field description: smth like  "Requested size of "state" partition(in percents of its capacity)

Reply:
done